molecule/shared/playbook.yml

```diff
@@ -0,0 +1,23 @@
+- name: converge
+  hosts: all
+  roles:
+    - role: passwordless_ssh
+      passwordless_ssh_public_key: "{{ role_path }}/tests/id_rsa.pub"
+      passwordless_ssh_private_key: "{{ role_path }}/tests/id_rsa"
+      passwordless_ssh_user: derp
+      passwordless_ssh_user_home: /home/derp
+  post_tasks: []
+
+- name: test host 0 connecting to host 1 without password
+  hosts: all[0]
+  tasks:
+    - name: try to ssh
+      command: ssh {{ hostvars[groups['all'][1]]['ansible_host'] }} ls -a /
+      become: true
+      become_user: derp
```

Comment:
Shouldn't this be replaced with `"{{ passwordless_ssh_user }}"` or `"{{ passwordless_ssh_become_user }}"`?

Reply:
This is in the molecule test playbook. Those variables are only in scope inside of the role. These are test values I'm using.